tests/lib/test-vip-request-block.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+class VIP_Request_Block_Test extends \WP_UnitTestCase {
+	/*
+	 * The $_SERVER headers that are used in this class to test
+	 * are defined in the tests/bootstrap.php file.
+	 */
+
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+
+		require_once __DIR__ . '/../../lib/class-vip-request-block.php';
+	}
+
+	public function test__error_raised_true_client_ip() {
+		VIP_Request_Block::ip( '2.2.2.2' );
+
+		// Asserting that no exception has been raised up to this point
+		$this->assertTrue( true );
```

Comment:
Instead of asserting that true is true, consider using https://phpunit.readthedocs.io/en/7.5/annotations.html#doesnotperformassertions annotation instead, to avoid the risky test.